save-frontend/src/main/kotlin/com/saveourtool/save/frontend/components/topbar/TopBarUrlSplits.kt

```diff
@@ -0,0 +1,85 @@
+@file:Suppress("FILE_NAME_MATCH_CLASS")
+
+package com.saveourtool.save.frontend.components.topbar
+
+import com.saveourtool.save.frontend.utils.TopBarUrl
+import com.saveourtool.save.utils.URL_PATH_DELIMITER
+
+import csstype.ClassName
+import history.Location
+import react.FC
+import react.Props
+import react.dom.aria.AriaCurrent
+import react.dom.aria.ariaCurrent
+import react.dom.aria.ariaLabel
+import react.dom.html.ReactHTML.a
+import react.dom.html.ReactHTML.li
+import react.dom.html.ReactHTML.nav
+import react.dom.html.ReactHTML.ol
+import react.router.dom.Link
+
+val topBarUrlSplits = topBarUrlSplits()
+
+/**
+ * [Props] of the top bar url splits component
+ */
+external interface TopBarUrlSplitsProps : Props {
+    /**
+     * Is location
+     */
+    var location: Location
+}
+
+/**
+ * Displays the url and its division by "/"
```

Comment:
"Division" is unclear here.

I'd put this as

```suggestion
 * Displays the URL split with "/".
```

&mdash; but this is still hard to understand. Please clarify and add more context information.

Reply:
Done [faec44c](https://github.com/saveourtool/save-cloud/pull/1540/commits/faec44ca7ee13b63b5e76ba2be295039cdb7c3e1)